ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/stores/SandboxIndexAndWriteService.scala

```diff
@@ -101,7 +103,7 @@ object SandboxIndexAndWriteService {
       templateStore: InMemoryPackageStore)(implicit mat: Materializer) = {
```

Comment:
Seems like this method doesn't need the `templateStore` parameter anymore and the parameter can be removed.